src/server/game/Spells/Auras/SpellAuras.cpp

```diff
@@ -2039,6 +2046,34 @@ void Aura::TriggerProcOnEvent(uint8 procEffectMask, AuraApplication* aurApp, Pro
         Remove();
 }
 
+void Aura::HeartbeatResistance(uint32 diff, Unit* caster)
+{
+    Unit* target = GetOwner()->ToUnit();
+
+    if (!target || !caster)
+        return;
+
+    if (m_heartBeatTimer <= diff)
+    {
+        if (target->GetTypeId() == TYPEID_UNIT && caster->GetTypeId() == TYPEID_PLAYER)
+        {
+            SpellSchoolMask schoolMask = m_spellInfo->GetSchoolMask();
+            uint32 resistance = schoolMask != SPELL_SCHOOL_MASK_NORMAL ? target->GetResistance(GetFirstSchoolInMask(schoolMask)) : 0;
```

Comment:
if schoolMask == SPELL_SCHOOL_MASK_NORMAL is a case that shouldn't be handled, check that on start of the method and do an early return

Reply:
Just don't set heartbeat timer for auras with mask `NORMAL`, done